antelope_contracts/contracts/erc20/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ list(APPEND SOURCES
     ${EXTERNAL_DIR}/silkworm/silkworm/core/rlp/decode.cpp
 )
 
-
+add_compile_definitions(ANTELOPE)
```

Comment:
This could probably even be moved into the `antelope_contracts/contracts/CMakeLists.txt` so it is shared across all the Antelope contracts we build.